nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -547,14 +548,25 @@ function AddSubscriberDetails(props: DialogProps) {
             state: subscriber.state,
             sub_profile: subscriber.dataPlan,
           },
-        });
+        };
         successCountRef.current = successCountRef.current + 1;
```

Comment:
You can get rid of successCountRef

Reply:
It shouldn't be removed because successCountRef is used in the Dialog content in calculating the value for LinearProgressWithLabel.

Reply:
I have removed it